### PR TITLE
REF: Simplify "Inline function" refactoring dialog

### PR DESCRIPTION
**Pull request by dima74**
Remove options "Search in comments" and "Search for text occurrences" from "Inline function" and "Inline variable" dialogs. These options don't affect anything right now and in general are not very useful for these particular refactorings. 

|                                                  Before                                                   |                                                   After                                                   |
|:---------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------:|
|  | image |

changelog: Simplify "Inline function" refactoring dialog

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt [hunk 1]
```diff
@@ -73,4 +73,6 @@ class RsInlineFunctionDialog(
 
     override fun getHelpId(): String =
         "refactoring.inlineMethod"
+
+    override fun isInlineThis(): Boolean = false
```
Comment (ortem) [inline src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt:77]: Does this change something in the UI or behaviour? Maybe we could add a comment here

**Review by ortem — COMMENTED** [review]
Do I understand right that now there is no option "Don't show this dialog in the future" to save the refactoring setting?

Let's also add two screenshots "before" and "after"  to make it more clear what is actually changed in the UI.

File: src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt (hunk 1 above)
Comment (dima74) [reply src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt]: I removed this method, it was accidentally overwritten

**Comment by dima74** [comment]
> Do I understand right that now there is no option "Don't show this dialog in the future" to save the refactoring setting?

Yes, this option was also removed to be consistent with other languages. Instead they save last used option of refactoring (so e.g. if you choose "inline all + keep", then next time this option in dialog will be selected). I plan to support this in future PRs

**Review by ortem — APPROVED** [review]
OK, that sounds reasonable, thanks for clarifying!

bors r+

**Pull request merged**